seatunnel-ci-tools/src/test/java/org/apache/seatunnel/api/ConnectorOptionCheckTest.java

```diff
@@ -120,7 +120,11 @@ public void checkConnectorOptionExist() {
                                                                         || extendedType
                                                                                 .getNameAsString()
                                                                                 .equals(
-                                                                                        "IncrementalSource")) {
+                                                                                        "IncrementalSource")
+                                                                        || extendedType
+                                                                                .getNameAsString()
+                                                                                .equals(
+                                                                                        "BaseMultipleTableFileSink")) {
```

Comment:
please also add `BaseFileSink` and `BaseFileSource` extend class check. 
- https://github.com/apache/seatunnel/blob/dev/seatunnel-connectors-v2/connector-file/connector-file-cos/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/cos/sink/CosFileSink.java
- https://github.com/apache/seatunnel/blob/dev/seatunnel-connectors-v2/connector-file/connector-file-cos/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/cos/source/CosFileSource.java